en/gps_compass/rtk_gps.md

```diff
@@ -30,6 +30,7 @@ Device | GPS | Compass | [DroneCAN](../dronecan/index.md) | [GPS Yaw](#configuri
 [CubePilot Here+ RTK GPS](../gps_compass/rtk_gps_hex_hereplus.md) | M8P | HMC5983 | | |
 [CubePilot Here3 CAN GNSS GPS (M8N)](https://www.cubepilot.org/#/here/here3) | M8P | ICM20948 | &check; | |
 [Drotek SIRIUS RTK GNSS ROVER (F9P)](https://store-drotek.com/911-sirius-rtk-gnss-rover-f9p.html) | F9P | RM3100 | | [Dual F9P][DualF9P] |
+[DATAGNSS GEM1305 RTK Receiver][DATAGNSS GEM1305 RTK] | CYNOSURE IV | &cross; | | &cross; |
```

Comment:
What is the right name for the GNSS part? I.e. normally I put FP9, but here I put CYNOSURE IV - is FP9 correct?

Reply:
we don't use ublox F9P.
The chipset is from Allystar , which is called TAU951M series.

Reply:
```suggestion
[DATAGNSS GEM1305 RTK Receiver][DATAGNSS GEM1305 RTK] | TAU951M | &cross; | | &cross; |
```